### ISSUE TITLE
BUG: Strange error with "Sodium/Iris mod", The fire texture looks bugged.

### ISSUE THREAD

**Issue by BLATITUMC** [haubna/PhysicsMod#871]
**Describe the bug**
When entering a normal world, and entering lava, or interacting with a source of fire, the first-person animation is blurred and with certain errors, this only happens when activating any shader, and when deactivating Physics mod everything returns to the same. normal.

**To Reproduce**
.

**Screenshots**
![2024-04-14_17 53 51](https://github.com/haubna/PhysicsMod/assets/167039869/a78deb11-c638-46b0-a1d2-7973f911c43b)
m.

Minecraft Version: <1.20.1>
Physics Mod Version: <3.0.9>

**Comment by haubna** [comment]
Please update to the latest version of the mod and let me know if the problem still persists.

**Comment by BLATITUMC** [comment]
The problem persists, and the Mod version is "3.0.13" (1.20.1)
------
![Minecraft_ 1 20 1 - Singleplayer 15_04_2024 07_31_03 a  m](https://github.com/haubna/PhysicsMod/assets/167039869/9259548c-334a-4639-821b-dc56b34695fb)

**Comment by BLATITUMC** [comment]
Sorry for waiting[image: Minecraft_ 1.20.1 15_04_2024 03_24_01 p. m..png]
, I already installed the most recent version (3.0.13) from your Patreon,
and the error persists, I also noticed another error, in the textures of
the smoke particles from the campfires, the sea particles, and others
particles, here I leave you some screenshots:
[image: Minecraft_ 1.20.1 15_04_2024 03_23_34 p. m..png][image: Minecraft_
1.20.1 15_04_2024 03_23_46 p. m..png]

**Comment by haubna** [comment]
I will have a look at it, thanks for the report!

**Comment by BLATITUMC** [comment]
Yep, 👍

**Comment by haubna** [comment]
I wasn't able to reproduce it with Physics Mod 3.0.14 and Iris 1.7.0. Can you confirm that this is fixed in that version?

**Comment by BLATITUMC** [comment]
I´ll check and let you know

**Comment by BLATITUMC** [comment]
The bug with the particle textures was solved, but the fire texture bug
persists, another piece of information that could be useful to you: the
shaders that I use are ComplementaryShaders v4.7.2.
Another fact that I just discovered is that the mod's resource pack is NOT
compatible, even if it has the correct mod version.
If you want to know more about the texture packs or mods I use, you can let
me know. 👍
[image: Minecraft_ 1.20.1 - Un jugador 01_05_2024 11_44_03 p. m..png][image:
Minecraft_ 1.20.1 - Un jugador 01_05_2024 11_43_26 p. m..png][image:
Minecraft_ 1.20.1 - Singleplayer 01_05_2024 11_55_45 p. m..png]

**Comment by BLATITUMC** [comment]
Something I didn't put in by accident: the version it asks for simply
doesn't appear in the mods, even though it's in the Mods folder.

[image: image.png]

**Comment by haubna** [comment]
Yeah a list of all active mods would be helpful. I wasn't able to reproduce this with Physics Mod 3.0.14 + Iris 1.7.0 + Sodium 0.5.8. I've tried to simply jump into lava.

**Comment by BLATITUMC** [comment]
Yes, here are the mods that I regularly use:
Zoomfy
wthit
WorldEdit (creative mode only)
Trinkets
Traveler´s Backpack
Toolstats
Terralith
Sodium
Indium
Iris
Continuity
Sit mod
ResolutionControl +
Repurposed Structures
Promenade
Presence Footsteps
Pottery
Particle Rain
NotEnoughAnimations
Nature´s Compass
Naturalist
mOres
Mod Menu
MmmMmmMmm (dummy mod)
Mmm Indicators (alternative)
LambDynamicLights
JEI
Jump Over Fences
JourneyMap
ItemPhysic
Immersive Structures
Immersive Aircraft
Heart & Home
Geophilic
GeckoLib 4
Framework
FPS-Display
FirstPerson
FallingLeaves
Every Compat
Entity Texture Features (For FA)
Fresh Animations (FA)
Entity Model Features (FA)
EatingAnimation
CustomSkinLoader
Crossbow expansion
Camera Overhaul
Botany Pots
Bookshelf
Better Third Person
Better Nether
Better end
ACLib
Bad Packets
Architectury
AppleSkin
AnotherFurniture
Amendments
Additional Structures
PhysicsMod
 Some I only activate in the creative or test worlds, to learn about mods,
etc.
Here are the Resource Packs:
Physics Mod (Integrated) (Incompatible)
BetterLeaves
FreshAnimations
Eating Animation Support (integrated)
Moonlight Mods Dynamic Assets (integrated)
GUI RP
Glass Pane Culling Fix (integrated)
Default Connected Textures (Integrated)
Default Sound Pack (PresenceFootsteps)
Fabric API/Mods
Those would be the mods and RP that I use, I already tried removing all of
them except physicsmod and it still doesn't solve the bug, I hope it helps
you. 👍

**Comment by haubna** [comment]
Are you on M1 Mac hardware?

**Comment by BLATITUMC** [comment]
No, I have Windows 10

**Comment by haubna** [comment]
Have you changed any specific sodium/iris setting? Could you try it with only physics mod + iris + sodium and make sure to delete the old config files so that you have a fresh install.

**Comment by BLATITUMC** [comment]
Sure, I'll let you know.

**Comment by BLATITUMC** [comment]
[image: Minecraft_ 1.20.1 - Singleplayer 02_05_2024 06_01_53 p. m..png]
[image: Minecraft_ 1.20.1 - Singleplayer 02_05_2024 06_02_26 p. m..png]
[image: image.png]

**Comment by haubna** [comment]
Your screenshots aren't being displayed on github

**Comment by BLATITUMC** [comment]
Oh, sorry, I was watching it on Jmail, here are those same images re-uploaded.

![Minecraft_ 1 20 1 - Singleplayer 02_05_2024 06_02_26 p  m](https://github.com/haubna/PhysicsMod/assets/167039869/d03c8ab5-cb71-4a7a-ad3e-f5a51303d024)
![Minecraft_ 1 20 1 - Singleplayer 02_05_2024 06_01_53 p  m](https://github.com/haubna/PhysicsMod/assets/167039869/1bd32c12-b0a6-491b-aacf-f396c1b82842)

**Comment by haubna** [comment]
So this is only with Sodium + Iris + Physics Mod on Fabric with completely fresh configs?

**Comment by BLATITUMC** [comment]
Yes

**Comment by BLATITUMC** [comment]
I don't know if it has anything to do with it, but I use TLauncher, but I
discarded that idea, because when I tried it on my brother's account the
same thing happened.
